bigframes/core/compile/compiled.py

```diff
@@ -1069,7 +1091,15 @@ def _to_ibis_expr(
         return table
 
     def filter(self, predicate: ex.Expression) -> OrderedIR:
-        if any(map(is_window, map(self._get_ibis_column, predicate.unbound_variables))):
+        if any(
```

Comment:
Probably can be replaced by a simple for loop too...

Reply:
converted to loop